<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Hit &amp; Blow</string>
  </property>
  <widget class="QLabel" name="digit_0">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>100</width>
     <height>143</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="digit_1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>100</width>
     <height>143</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="digit_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>100</width>
     <height>143</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="digit_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>100</width>
     <height>143</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="arrow_down">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_try">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>235</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Try!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>- Control -</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>A, D : Select digit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>265</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>W, S : Select card</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Try Count:</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_try_count">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>30</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txt_log">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>400</width>
     <height>160</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="toolTip">
    <string>Log</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>100</width>
     <height>160</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="toolTip">
    <string>Memo</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_max_try_count">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>510</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_new_game">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>510</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>New Game</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Max. Try Count</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>510</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Max. Color</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_max_color">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>510</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>4</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_extra_deck">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>550</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Extra Deck</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>F: Select color</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>- Rule -</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>265</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hit: number + position + color</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Blow: number only or number + position</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txt_playlist">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>200</width>
     <height>235</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="toolTip">
    <string>Playlist</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_play">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>370</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Play</string>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>370</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Next song</string>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_continuous">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>410</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>If you check it, the playlist won't stop playing</string>
   </property>
   <property name="text">
    <string>Continuous</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_shuffle">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>410</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>If you check it and play the playlist, the song will shuffle</string>
   </property>
   <property name="text">
    <string>Shuffle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_browse">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Browse music file</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_prev">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>370</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Previous song</string>
   </property>
   <property name="text">
    <string>Prev</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>0</y>
     <width>20</width>
     <height>600</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txt_current_song">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>330</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="toolTip">
    <string>Current song</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_volume">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>480</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Volume</string>
   </property>
   <property name="pageStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>460</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>10</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Clear the playlist</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pause">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>370</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Pause</string>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stop">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>370</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Play</string>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_resume">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>370</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Resume</string>
   </property>
   <property name="text">
    <string>Resume</string>
   </property>
  </widget>
  <zorder>btn_resume</zorder>
  <zorder>btn_pause</zorder>
  <zorder>digit_0</zorder>
  <zorder>digit_1</zorder>
  <zorder>digit_2</zorder>
  <zorder>digit_3</zorder>
  <zorder>arrow_down</zorder>
  <zorder>btn_try</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>txt_try_count</zorder>
  <zorder>txt_log</zorder>
  <zorder>textBrowser</zorder>
  <zorder>spin_max_try_count</zorder>
  <zorder>btn_new_game</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>spin_max_color</zorder>
  <zorder>chk_extra_deck</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>txt_playlist</zorder>
  <zorder>btn_play</zorder>
  <zorder>btn_next</zorder>
  <zorder>chk_continuous</zorder>
  <zorder>chk_shuffle</zorder>
  <zorder>btn_browse</zorder>
  <zorder>btn_prev</zorder>
  <zorder>line</zorder>
  <zorder>txt_current_song</zorder>
  <zorder>slider_volume</zorder>
  <zorder>label_11</zorder>
  <zorder>btn_clear</zorder>
  <zorder>btn_stop</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
